build.yaml

```diff
@@ -1,10 +1,11 @@
 targets:
   $default:
     builders:
-      mockito:mockBuilder:
-        # Scope only to files declaring mocks, for performance.
-        generate_for:
-          - test/mockito.dart
+      test_html_builder:
+        options:
+          templates:
+            "test/templates/html_template.html":
+              - "test/**_test.dart"
```

Comment:
Moved to using test_html_builder to make it easier to swap js files that were being used for testing with react 17 and 18 in ci